package/nmea_protocol/src/nmea_router.yml

```diff
@@ -0,0 +1,14 @@
+
+name: NMEA_ROUTER
+ports:
+  - name: NMEA_PORT_FIRMWARE
+    pub_addr: "@tcp://127.0.0.1:44010"
+    sub_addr: "@tcp://127.0.0.1:44011"
+    forwarding_rules:
+      - dst_port: NMEA_PORT_EXTERNAL
+        filters:
+          - { action: ACCEPT }
```

Comment:
With what goal? FYI most filters contain an action and a prefix. NMEA is pretty trivial.